### ISSUE TITLE
[Youtube Music] "Artist 1 & Artist 2" turns to "Artist 1, Artist 2"

### ISSUE THREAD

**Issue by abbyfour** [web-scrobbler/web-scrobbler#2806]
**Describe the bug**
Web scrobbler alters the metadata from "Artist 1 & Artist 2" to "Artist 1, Artist 2"

**How to reproduce**
1. On Youtube Music, play an artist with an & in their name (I can't say for sure that it happens for *all* artists, but I've noticed it quite frequently). Here are some artists that definitely do it:
- Freddie Gibbs & Madlib
- Zack Fox & Kenny Beats
- Disclosure & Slowthai

**Expected behavior**
I expect the metadata to remain unchanged. (or, alternatively if this is intentional, I should be able to turn this off)

**Screenshots**
![image](https://user-images.githubusercontent.com/32883726/113061293-aa03ce80-9166-11eb-962d-dad0c8132195.png)
![image](https://user-images.githubusercontent.com/32883726/113061321-b425cd00-9166-11eb-94d6-d3f043b53491.png)

**Debug logs**
https://pastebin.com/WcB2atW0

**Environment (please complete the following information)**
- Mac
- Chrome
- Web scrobbler 2.41.0

**Additional context**
This didn't use to happen, I think I started noticing it about 1-2 weeks ago?

I'm also confident this isn't a Last.fm redirect issue, because my Discord bot which integrates with the last.fm api does not have this issue

**Comment by alexesprit** [comment]
It's intentional.

**Comment by jivison** [comment]
@alexesprit  Would it be possible to have the option to disable it? I find it super annoying because all my scrobbles are under the tag with the `&`, and often album covers and metadata is missing for the `Artist, Artist` artists on Last.fm

**Comment by flickur** [comment]
Yeah if it's intentional it's simply wrong. I don't think i've ever seen "artist1, artist2" as correct artist tag, "artist1 & artist2" is very common.

**Comment by rob9315** [comment]
original issue should be resolved:
<details><summary>Full Tac & Lil Mariko</summary>
<img width="1456" height="972" alt="Image" src="https://github.com/user-attachments/assets/5c9a3e6f-ebbd-48c6-b1af-b6f9305aa2ad" />
</details>
example taken from the pastebin
